src/test/java/edu/mit/dos/object/ObjectServiceTest.java

```diff
@@ -37,7 +44,8 @@ public void setup(){
 
     @Test
     public void testGet() throws Exception {
-        mockMvc.perform(get("/object").param("oid", "123877664"))
+        when(objectJpaRepository.findByOid(1222)).thenReturn(new DigitalObject());
+        mockMvc.perform(get("/object").param("oid", "1222"))
```

Comment:
This should also confirm that the returned response object (in JSON representation) does have the oid 1222. 

Something like:

`andExpect(content.json("{'oid':1222}))`